cryodrgn/commands/abinit_het.py

```diff
@@ -1055,39 +1065,43 @@ def flog(msg):  # HACK: switch to logging module
                     out_poses,
                 )
 
-    # save model weights and evaluate the model on 3D lattice
```

Comment:
A minor bug with loading a saved model and trying to re-run it with the same epochs (so essentially the code wouldn't do anything), would have resulted in an error, since `epoch` is undefined (and a further `ZeroDivisionError` when trying to write a message). So this is now protected by `if epoch is not None`. Same change for `abinit_homo`.

These are the kinds of checks that will be flagged by the upcoming type-checking stuff we plan to put in.